lib/exposes.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
```

Comment:
We can use something like builder pattern here, and with this it we don't have to make this endless combinations of possible features.
https://gist.github.com/nurikk/00b55d6c3dc38ad45878a6d21fbeac88